chat-client/package.json

```diff
@@ -17,9 +17,9 @@
         "package": "webpack"
     },
     "dependencies": {
-        "@aws/chat-client-ui-types": "0.x.x",
-        "@aws/language-server-runtimes-types": "0.x.x",
-        "@aws/mynah-ui": "^4.11.x"
+        "@aws/chat-client-ui-types": "^0.0.6",
```

Comment:
I think we shouldn't change these values from `x.x` suffix - iirc Ivan discovered that when using caret (^) syntax in npm will result in not upgrading patch version. This effectively pins version to `0.0.6` untl manually upgraded

Reply:
0.x.x has the same effect as caret since the jump from 0.0.6 to 0.0.7 is also considered a major version. We no longer need the 0.x.x type values and this makes it easier and more explicit when we want to bump  to newer versions

Reply:
ah, ok, I must have remembered that wrong